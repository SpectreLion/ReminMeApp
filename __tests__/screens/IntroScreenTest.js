import 'react-native';
import React from 'react';
//IMPORT COMPONENTS
import introscreen from '../../src/screens/introscreen';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

describe('introscreen component',()=>{

  it('should run a test to check', ()=>{
    const introscreencomponent = renderer.create(<introscreen/>).toJSON();
    expect(introscreencomponent).toMatchSnapshot();
  });
});
