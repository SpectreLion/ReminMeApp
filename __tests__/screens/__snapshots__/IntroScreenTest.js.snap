// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`introscreen component should run a test to check 1`] = `<introscreen />`;
